README.md

```diff
@@ -123,31 +121,32 @@ public static SootClass loadClass(String name, boolean main, String module) {
 }
 
 ```
-
+ModuleUtil.module_mode() helps you check if you have modules enabled in Soot. This is done based on whether the module path is set using the Options class.
 
 ### Example Configurations: Java 8, Java >= 9 Classpath, Java >= 9 Modulepath
 
 ```.java
 
-if(java < 9 ) {
+if(java < 9 ) { // when you want Java < 9 for Soot and you are not using Soot on modules
     Options.v().set_prepend_classpath(true);
     Options.v().set_process_dir(Arrays.asList(applicationClassPath().split(File.pathSeparator)));
-    Options.v().set_claspath(sootClassPath();
+    Options.v().set_soot_classpath(sootClassPath());
 }
 
-if(java >= 9 && USE_CLASSPATH){
+if(java >= 9 && USE_CLASSPATH) { // when you want Java >= 9 for Soot and you are not using Soot on modules
```

Comment:
when you have a target benchmark which is for Java >= 9 and want no module support